workers/unity/Packages/com.improbable.gdk.playerlifecycle/Systems/PlayerCreation/SendCreatePlayerRequestSystem.cs

```diff
@@ -31,43 +35,54 @@ protected override void OnCreateManager()
                 ComponentType.ReadOnly<WorkerEntityTag>(),
                 ComponentType.ReadOnly<OnConnected>()
             );
+        }
 
-            playerSpawnGroup = GetComponentGroup(
-                ComponentType.ReadOnly<ShouldRequestPlayerTag>()
-            );
+        public void QueuePlayerCreationRequest(Vector3 spawnPosition = default, byte[] serializedArguments = null)
```

Comment:
This doesn't seem to queue anything

Reply:
Presumably we only allow a client to request a single player?

Reply:
Yes, clients can only request a single player

The `Queue` name was initially `Trigger`, but it doesn't exactly trigger a player creation request

perhaps it could renamed to `RequestPlayerCreation`?

Reply:
Request would be more in keeping with what it does. Except I would expect that to do something immediately and that multiple calls to Request... would not cancel each other out.